src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -709,6 +712,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
 
         stats.setNumberOfAssessmentLocks(numberOfAssessmentLocks);
 
+        start = System.currentTimeMillis();
+        // 10ms - fast
```

Comment:
I do not quite understand what these comments meant to say 😄 Is it the baseline, you can compare against what the log reports?